tests/regression/71-localTraces/33-nondeterministc-loop-exit.c

```diff
@@ -0,0 +1,19 @@
+// NONTERM
+// PARAM:  --set "ana.activated[+]" localTraces --set solver "WLLocTrac" --set
+// warn_at "early" --set verify false
```

Comment:
I'm pretty sure our scripts don't allow `PARAM` to be split onto multiple lines like that.